src/clr-angular/popover/dropdown/dropdown-menu.ts

```diff
@@ -86,4 +103,19 @@ export class ClrDropdownMenu extends AbstractPopover {
         break;
     }
   }
+
+  private focusHandler: DropdownFocusHandler;
+  @ContentChildren(FocusableItem) items: QueryList<FocusableItem>;
+
+  ngAfterContentInit() {
+    this.focusHandler.container = this.el.nativeElement;
+    this.items.changes.subscribe(() => this.focusHandler.addChildren(this.items.toArray()));
+    // I saw this on GitHub as a solution to avoid code duplication because of missed QueryList changes
+    this.items.notifyOnChanges();
```

Comment:
Should this go inside of the changes handler on line 112?

Reply:
No. Actually that line is what's causing the change event to fire.